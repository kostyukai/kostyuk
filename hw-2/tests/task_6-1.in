16
23.51
