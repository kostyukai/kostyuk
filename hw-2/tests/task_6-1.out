OF=0
SF=1
CF=1
ZF=0
